dockstore-webservice/src/main/java/io/dockstore/webservice/helpers/ZenodoHelper.java

```diff
@@ -156,6 +158,22 @@ public static void registerZenodoDOIForWorkflow(ApiClient zenodoClient, String d
 
         Deposit publishedDeposit = publishDepositOnZenodo(actionsApi, depositionID);
 
+        Object links = publishedDeposit.getLinks();
+        String conceptDoiUrl = (String)((LinkedHashMap)links).get("conceptdoi");
```

Comment:
it doesn't seem like there is a point to cast to `LinkedHashMap` in this case if only the get method is needed and it can break . 
Try just casting to `Map` or maybe `Map<String, String>`

Better yet, I think we control the `links` model, so we should probably just make it a `Map<String, String>` or a regular object with a conceptdoi field  in the first place to avoid casting at all.

Reply:
Not sure what you mean here...could you explain?
![Screen Shot 2019-09-29 at 9 25 22 PM](https://user-images.githubusercontent.com/3650921/65849585-07b02400-e300-11e9-87e6-3a883a903cef.png)

Reply:
As I understand it, "publishDeposit" is a swagger generated object from a swagger.yaml that we control. Rather than having that generated code return a simple object when calling `getLinks`, it should really be improved to return an object with typesafe properties. 

i.e. If we know links includes a string called conceptdoi, just have the generated code for links return a string and avoid casting in the first place

Reply:
Done